release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Added the 'create meeting' action to get screenshots from a dynamic analysis
```

Comment:
Is `create meeting` a typo? Or a placeholder for something? Shouldn't it be `get screenshots`?

Reply:
yes it was a typo